app/src/main/java/org/mozilla/fenix/sync/SyncedTabsViewHolder.kt

```diff
@@ -0,0 +1,79 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.fenix.sync
+
+import android.view.View
+import androidx.recyclerview.widget.RecyclerView
+import kotlinx.android.synthetic.main.sync_tabs_list_item.view.*
+import kotlinx.android.synthetic.main.view_synced_tabs_group.view.*
+import mozilla.components.browser.storage.sync.Tab
+import org.mozilla.fenix.R
+import org.mozilla.fenix.sync.SyncedTabsAdapter.AdapterItem
+import org.mozilla.fenix.utils.Do
+
+sealed class SyncedTabsViewHolder(itemView: View) : RecyclerView.ViewHolder(itemView) {
+
+    abstract fun <T : AdapterItem> bind(item: T, interactor: (Tab) -> Unit)
+
+    class TabViewHolder(itemView: View) : SyncedTabsViewHolder(itemView) {
+
+        init {
+            setupMenu()
+        }
+
+        override fun <T : AdapterItem> bind(item: T, interactor: (Tab) -> Unit) {
+            bindTab(item as AdapterItem.Tab)
+
+            itemView.setOnClickListener {
+                interactor(item.tab)
+            }
+        }
+
+        private fun bindTab(tab: AdapterItem.Tab) {
+            val active = tab.tab.active()
+            itemView.synced_tab_layout.titleView.text = active.title
+            itemView.synced_tab_layout.urlView.text = active.url
+
+            itemView.synced_tab_layout.loadFavicon(active.url)
+
+            // TODO download and set icon image.
```

Comment:
Aren't we already setting an icon image above this line?

Reply:
Ah okay nice I think these are fairly new designs. For reference:
<img width="382" alt="Screen Shot 2020-05-27 at 9 18 16 PM" src="https://user-images.githubusercontent.com/25442310/83087592-7936a280-a046-11ea-8b32-31ac48b2af37.png">
Not entirely sure if these are final, but sure let's remove the favicon for now